rest-service/manager_rest/blueprints_manager.py

```diff
@@ -890,29 +873,9 @@ def _get_resolver(self):
         return current_app.resolver
 
     def _update_import_resolver_in_app(self, context):
-        resolver = DefaultImportResolver()
-        resolver_class_path, params = self._get_resolver_section(context)
-        if resolver_class_path:
-            try:
-                resolver = get_class_instance(resolver_class_path, params)
-            except RuntimeError, ex:
-                raise ResolverInstantiationError(
-                    'Failed to instantiate resolver ({0}). {1}'
-                    .format(resolver_class_path, str(ex)))
-        else:
-            # using the default resolver
-            if params:
-                # using custom rules for the default resolver
-                try:
-                    resolver = DefaultImportResolver(
-                        rules=params.get(DEFAULT_RESLOVER_RULES_KEY))
-                except DefaultResolverValidationException, ex:
-                    raise ResolverInstantiationError(
-                        'Wrong parameters ({0}) configured for '
-                        'the default resolver ({1}): {2}'
-                        .format(params,
-                                DefaultImportResolver.__name__, str(ex)))
-        current_app.resolver = resolver
+        resolver_section = self._get_resolver_section(context)
+        current_app.resolver = dsl_parser_utils.create_import_resolver(
+            resolver_section)
```

Comment:
regarding update_provider_context (not modified here) - why does it contain _update_import_resolver_in_app? should it be separate? seems like the meaning is not the same